### ISSUE TITLE
Create Homebrew formula for Fontist

### ISSUE THREAD

**Issue by strogonoff** [fontist/homebrew-fontist#1]
It could be a shell file that checks for Ruby and other dependencies, and runs the installation.

A single command that fetches that file over HTTPS and pipes into `/bin/bash` could be given on landing page, similar to what Homebrew does. (Note: raises stakes as far as landing page security.)

**Comment by ronaldtse** [comment]
I suppose we can just do something like this:
https://formulae.brew.sh/formula/asciidoctor

Asciidoctor is a Ruby gem but also available via Homebrew:
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/asciidoctor.rb

**Comment by alemorozov** [comment]
Very clear example, can do the same for fontist.

**Comment by abunashir** [comment]
@alexeymorozov: Any update on this one? Did you already implemented it?

**Comment by alemorozov** [comment]
@abunashir Nope, currently no work is done on this task. It's only scheduled.
